### ISSUE TITLE
Rework Navigation in Arctosdb.org

### ISSUE THREAD

**Issue by mkoo** [ArctosDB/documentation-wiki#153]
Phyllis' suggestions here:
[Recommended Outline for Arctosdb.docx](https://github.com/ArctosDB/documentation-wiki/files/3074284/Recommended.Outline.for.Arctosdb.docx)

Obviously also part of Home page #152

**Comment by sharpphyl** [comment]
My suggestions have probably fallen out of date, but hope they can be a conversation starter.  Happy to help where I can.

**Comment by mkoo** [comment]
True, as I have actually changed some of the navigation based on your thoughts so thanks!
Also adding here, possibly adding new page or pages to highlight Arctos collections and institutions.
